README.md

```diff
@@ -139,7 +139,7 @@ Now you can either
 
 **Note:** After finishing testing, please clean up after yourself:
 
-- Execute `./test/tools/integration/cleanup_machines.sh` while the machine-controller is still running
+- Execute `./hack/cleanup_machines.sh` while the machine-controller is still running
 - Execute `make e2e-destroy` to clean up the test control plane
```

Comment:
This target doesn't exist any longer. We should update the README because it's also mentioning `make e2e-cluster` above.

Reply:
I've removed this portion completely and put a `WIP` in there. We can work on this later in a separate PR since it requires some changes to the scripts.